### PR TITLE
Add more os, arch checks to the shell installer

### PR DESCRIPTION
**Pull request by nehaljwani**
Fixes https://github.com/conda/constructor/issues/164
Fixes https://github.com/ContinuumIO/anaconda/issues/1126

### CODE REVIEW

File: constructor/header.sh [hunk 1]
```diff
@@ -213,6 +213,57 @@ then
     fi
 #endif
 
+#if ppc64le
+    if [ "$(uname -m)" != "ppc64le" ]; then
+        printf "WARNING:\\n"
+        printf "    Your machine hardware does not to appear be Power8 (little endian), \\n"
```
Comment (mingwandroid) [inline constructor/header.sh]: Please change:

> does not to appear be

.. to:

> does not appear to be

(in all of these)

Comment (mingwandroid) [inline constructor/header.sh]: Can we also have a way to skip this check? If `binfmt_misc` supports the architecture of the installer then I would not like to be forced to interactively confirm this, also what about when using `--force`? That usually means "no user interaction", should this abort automatically in this case, then do we need `--really-force` to indicate we don't care that `uname` is mismatched?

Comment (nehaljwani) [reply constructor/header.sh]: All of these checks are skipped if -b is specified.

Comment (nehaljwani) [reply constructor/header.sh]: Fixed in the commit below.

**Comment by mingwandroid** [comment]
Thanks Nehal

**Pull request merged**